Source/WebKit/Configurations/GPUService.xcconfig

```diff
@@ -25,6 +25,7 @@
 
 PRODUCT_NAME = com.apple.WebKit.GPU;
 PRODUCT_BUNDLE_IDENTIFIER = $(PRODUCT_NAME);
+
```

Comment:
nit: delete these blank lines when you have a chance

Reply:
Done!